src/vs/workbench/services/suggest/browser/media/suggest.css

```diff
@@ -117,8 +117,12 @@
 	touch-action: none;
 }
 
-.workbench-suggest-widget .monaco-list .monaco-list-row.focused {
-	color: var(--vscode-editorSuggestWidget-selectedForeground);
+/* Override any inherited background/foreground for focused row */
+.partial-selection.workbench-suggest-widget .monaco-list .monaco-list-row.focused {
```

Comment:
Could we retain the icon color for partial? The reason we change it is to guarantee contrast with the other background color:

![image](https://github.com/user-attachments/assets/9f161487-54d6-4d28-b660-10654acce926)

Reply:
Tried and it's not working 

<img width="738" alt="Screenshot 2025-06-17 at 6 14 57 PM" src="https://github.com/user-attachments/assets/89c66589-a0d1-4d50-9793-08a3b7f39dbf" />

Reply:
oh that is getting applied, but it's the wrong color. we want the terminal color, not sure how to specify that

Reply:
would need the id for the element to override it 🤯

Reply:
see `.monaco-list.list_id_25 .monaco-list-row.focused .codicon`

Reply:
I think this is actually inlined or something, @tyriar not sure how to fix it, could use your help

Reply:
<img width="551" alt="Screenshot 2025-06-17 at 7 59 22 PM" src="https://github.com/user-attachments/assets/d86fd064-d467-481d-88f9-088037743eee" />

This takes effect, but what we need is the appropriate terminal symbol icon color to not be overridden in the first place, bc we cannot dynamically get that here

```
#terminal .partial-selection.workbench-suggest-widget .monaco-list .monaco-list-row.focused .codicon {
	color:red;
}

```